XenModel/Messages.resx

```diff
@@ -11627,6 +11627,9 @@ Check your settings and try again.</value>
   <data name="UPDATES_WIZARD_CANNOT_MIGRATE_VM_LICENSE_REASON" xml:space="preserve">
     <value>Cannot migrate VM '{0}' due to license restrictions.</value>
   </data>
+  <data name="UPDATES_WIZARD_CANNOT_MIGRATE_VM_SUSPEND_REASON" xml:space="preserve">
+    <value>Cannot migrate VM '{0}', because it lacks the ability to suspend. Please ensure it has up-to-date drivers installed.</value>

```

Comment:
Suggestion. If it doesn't have tools, say "Cannot migrate VM '{0}', because it doesn't have up-to-date drivers installed" (but continue to offer shut down as the resolution). If it does have tools, but still can't migrate, say just "Cannot migrate VM '{0}'".